app/src/main/java/com/goldenraven/padawanwallet/data/tutorial/Tutorial.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020-2022 thunderbiscuit and contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the ./LICENSE file.
+ */
+
+package com.goldenraven.padawanwallet.data.tutorial
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.goldenraven.padawanwallet.ui.tutorials.TutorialElement
+
+@Entity(tableName = "tutorial_db")
+data class Tutorial(
+    @PrimaryKey(autoGenerate = false)
+    val id: Int,
+    val title: String,
+    val type: String,
+    val difficulty: String,
+    var completion: Int
```

Comment:
In this case completion refers to how many pages of the tutorial is completed, if completed == pages.size() it would mean that every page of the tutorial is completed